daml_dit_ddit/subcommand_release.py

```diff
@@ -94,6 +94,11 @@ def subcommand_main(force: bool, dry_run: bool):
 
     LOG.info(f'Releasing version {catalog.version} as {tag_name}.')
 
+    prerelease = "experimental" in catalog.tags
```

Comment:
You should also check the experimental flag:

https://github.com/digital-asset/daml-dit-api/blob/master/daml_dit_api/package_metadata.py#L44

Historically speaking, the `experimental` flag came before we had the more general notion of tags, and there still might be usages of the experimental flag. (Which is probably worth throwing a deprecated warning for, now that I think of it.)